kahuna/public/js/components/gr-panel/gr-panel.js

```diff
@@ -53,24 +59,47 @@ grPanel.controller('GrPanel', [
             });
         };
 
+        ctrl.updateUsageRights = () => {
```

Comment:
I'm really struggling to work out how to sensibly propagate the image changes back to the view. After we've saved we need to replace the image model in the search view and in the panel, and I don't know how to get a handle on those objects.